.github/workflows/proxy_plugin.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: Foreman Proxy Plugin
```

Comment:
Bikeshedding -- given the code projects are all named smart_proxy why choose Foreman Proxy here?

Reply:
I created this file based on `foreman_plugin.yml` and replacing it that way was quicker ;)

(also, you could argue the file itself should be called `foreman_proxy_plugin` or `smart_proxy_plugin`)

At the very end, I don't care either way :)

Reply:
Fair, `smart_proxy_plugin` would be more accurate and connect more cleanly.